app/controllers/api/v1x0/workflows_controller.rb

```diff
@@ -48,6 +49,13 @@ def update
       def workflow_params
         params.permit(:name, :description, :group_refs => [])
       end
+
+      def rbac_scope(relation)
+        access_obj = RBAC::Access.new('workflows', 'read').process
+        raise Exceptions::NotAuthorizedError, "Not Authorized to list workflows" unless access_obj.accessible? || access_obj.admin?
```

Comment:
check only `accessible?`. combine line 54 and 55 into one line. Maybe rubocop will no longer complain for code similarity.